src/interpreter/initialization.rs

```diff
@@ -40,41 +39,32 @@ where
         self.push_stack(tx.id().as_ref())
             .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
 
-        let zeroes = &[0; MAX_INPUTS as usize * (Color::LEN + WORD_SIZE)];
-        let ssp = self.registers[REG_SSP] as usize;
-
-        self.push_stack(zeroes)
-            .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
-
-        if tx.is_script() {
```

Comment:
Is there a reason to remove the `is_script()` checks here?

Reply:
because otherwise we wouldn't have the ability to track free balance of the inputs for create tx, which means they wouldn't be able to return change.